proto/rollup/v1/query.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package rollup.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "rollup/v1/rollup.proto";
+
+option go_package = "github.com/polymerdao/monomer/x/rollup/types";
+
+// Query defines all query endpoints for the rollup module.
+service Query {
+  // Params returns all rollup module parameters.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
```

Comment:
is this endpoint standard for other modules?

Reply:
Yep it follows the same format, here's the `Params` [endpoint](https://github.com/cosmos/cosmos-sdk/blob/99b4858a958c4986fe5eed367446c41b91a20ef6/x/bank/proto/cosmos/bank/v1beta1/query.proto#L71-L75) from `x/bank` as an example:

```
  // Params queries the parameters of x/bank module.
  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
    option (cosmos.query.v1.module_query_safe) = true;
    option (google.api.http).get               = "/cosmos/bank/v1beta1/params";
  }
```